5-Accessibility/accessible-form.html

```diff
@@ -29,16 +32,19 @@
         <fieldset>
             <legend>Please select the courses that you are interested in (required):</legend>
             <div>
-                <input type="checkbox" name="web" id="web">
+                <input type="checkbox" name="web" id="web" aria-required="true">
```

Comment:
In the `checkbox` input it's not necessary to put it as required. Remember that we talked about it in the last session.